functions/Invoke-DbaDbShrink.ps1

```diff
@@ -0,0 +1,345 @@
+            Exclude DBCC UPDATE USAGE for database.
+
+        .PARAMETER WhatIf
+            Shows what would happen if the command were to run.
+
+        .PARAMETER Confirm
+            Prompts for confirmation of every step. For example:
+
+            Are you sure you want to perform this action?
+            Performing the operation "Shrink database" on target "pubs on SQL2016\VNEXT".
+            [Y] Yes  [A] Yes to All  [N] No  [L] No to All  [S] Suspend  [?] Help (default is "Y"):
+
+        .PARAMETER EnableException
+            By default, when something goes wrong we try to catch it, interpret it and give you a friendly warning message.
+            This avoids overwhelming you with "sea of red" exceptions, but is inconvenient because it basically disables advanced scripting.
+            Using this switch turns this "nice by default" feature off and enables you to catch exceptions with your own try/catch.
+
+        .NOTES
+            Tags: Shrink, Database
+
+            Website: https://dbatools.io
+            Copyright: (C) Chrissy LeMaire, clemaire@gmail.com
+            License: MIT https://opensource.org/licenses/MIT
+
+        .LINK
+            https://dbatools.io/Invoke-DbaDatabaseShrink
+
+        .EXAMPLE
+            Invoke-DbaDatabaseShrink -SqlInstance sql2016 -Database Northwind,pubs,Adventureworks2014
+
+            Shrinks Northwind, pubs and Adventureworks2014 to have as little free space as possible.
+
+        .EXAMPLE
+            Invoke-DbaDatabaseShrink -SqlInstance sql2014 -Database AdventureWorks2014 -PercentFreeSpace 50
+
+            Shrinks AdventureWorks2014 to have 50% free space. So let's say AdventureWorks2014 was 1GB and it's using 100MB space. The database free space would be reduced to 50MB.
+
+        .EXAMPLE
+            Invoke-DbaDatabaseShrink -SqlInstance sql2014 -Database AdventureWorks2014 -PercentFreeSpace 50 -FileType Data -StepSizeMB 25
+
+            Shrinks AdventureWorks2014 to have 50% free space, runs shrinks in 25MB chunks for improved performance.
+
+        .EXAMPLE
+            Invoke-DbaDatabaseShrink -SqlInstance sql2012 -AllUserDatabases
+
+            Shrinks all databases on SQL2012 (not ideal for production)
+
+    #>
+    [CmdletBinding(SupportsShouldProcess = $true, ConfirmImpact = 'Low')]
+    param (
+        [parameter(Mandatory = $true, ValueFromPipeline = $true)]
+        [Alias("ServerInstance", "SqlServer")]
+        [DbaInstanceParameter[]]$SqlInstance,
+        [Alias("Credential")]
+        [PSCredential]$SqlCredential,
+        [Alias("Databases")]
+        [object[]]$Database,
+        [object[]]$ExcludeDatabase,
+        [switch]$AllUserDatabases,
+        [ValidateRange(0, 99)]
+        [int]$PercentFreeSpace = 0,
+        [ValidateSet('Default', 'EmptyFile', 'NoTruncate', 'TruncateOnly')]
+        [string]$ShrinkMethod = "Default",
+        [ValidateSet('All', 'Data', 'Log')]
+        [string]$FileType = "All",
+        [int]$StepSizeMB,
```

Comment:
I agree with @potatoqualitee on this one to drop the MB

Reply:
done 👍